tests/stubs/edge/handlerstub/module.go

```diff
@@ -67,5 +67,8 @@ func (hs *HandlerStub) Start() {
 		klog.Errorf("New upstream controller failed with error: %v", err)
 		return
 	}
-	upstream.Start()
+	if err := upstream.Start(); err != nil {
```

Comment:
same here, `func (dc *UpstreamController) Start()` return `nil` directly,  there is no chance to reach this code block here.

Reply:
Same as above.